lib/core/src/Cardano/Wallet/Primitive/Types/Tx/Gen.hs

```diff
@@ -125,12 +132,18 @@ txToTxWithoutId :: Tx -> TxWithoutId
 txToTxWithoutId Tx {..} = TxWithoutId {..}
 
 txWithoutIdToTuple :: TxWithoutId -> _
-txWithoutIdToTuple (TxWithoutId a1 a2 a3 a4 a5 a6) =
-    (a1, a2, a3, a4, a5, a6)
+txWithoutIdToTuple (TxWithoutId a1 a2 a3 a4 a5 a6 a7) =
+    (a1, a2, a3, a4, a5, a6, a7)
 
 tupleToTxWithoutId :: _ -> TxWithoutId
-tupleToTxWithoutId (a1, a2, a3, a4, a5, a6) =
-    (TxWithoutId a1 a2 a3 a4 a5 a6)
+tupleToTxWithoutId (a1, a2, a3, a4, a5, a6, a7) =
+    (TxWithoutId a1 a2 a3 a4 a5 a6 a7)
+
+genScriptValidation :: Gen ScriptValidation
+genScriptValidation = genericArbitrary
```

Comment:
Here you could also use `arbitraryBoundedEnum`, which will generate a uniform distribution across all possibilities.